src/components/progress-tracker/AuiProgressTrackerStep.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <li class="aui-progress-tracker-step" :class="{'aui-progress-tracker-step-current': active}" style="width:25%"><span><slot></slot></span></li>
```

Comment:
Please break this line as it's pretty long.

There's also a major problem with the styling here. `width: 25%` seems to only be working properly to up to 4 items. Calculating proper width (100% divided by number of items) here is not very convenient here as we'd have to count the steps (not that pleasant). What I found is a good heuristic here is to set width to 1%. As these items are in the table, they'll expand anyway. The only difference in the behaviour is that the columns in this case don't respect their `max-width`, but that's fine to me in this case.

Reply:
I already tested it, 25% seemed to be correct for an arbitray number of steps, but I will double check this. Thanks!

Reply:
I was testing it on Chrome, try with different window widths.